include/gazebo_mavlink_interface.h

```diff
@@ -18,17 +18,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include <vector>
```

Comment:
Are all these includes really needed? It looks like a collection of almost everything boost has to offer.

Reply:
Probably not.

Reply:
@arwagoner can you check if we really need all these includes?